lib/expressions.js

```diff
@@ -29,7 +29,10 @@ function primaryName(expression) {
   if (expression.name) {
     expression = expression.name;
   }
-  if (expression.indexOf('\'') === 0 || expression.match(/^[0-9]/)) {
+  if (expression.match(/^('|").*('|")$/)) { // string literal
+    return '';
+  }
+  if (expression.match(/^-?\d+/)) { // number literal
```

Comment:
Thanks for your review. I have used a slightly different now, which would also allow things like "-.5". Please note that things like 123e-5 or 0xFF are not supported now (but they are also not supported by Polymer (yet)).